shared/django_apps/legacy_migrations/migrations/0005_auto_20241112_2306.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 4.2.16 on 2024-11-12 23:06
+
+from django.db import migrations
+
+from shared.django_apps.migration_utils import RiskyRunSQL
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('legacy_migrations', '0004_auto_20231024_1937'),
+    ]
+
+    operations = [
+        RiskyRunSQL(
+            """
+            DROP TRIGGER IF EXISTS branch_update ON branches;
+            DROP FUNCTION IF EXISTS branches_update();
+            """
+        ),
+    ]
```

Comment:
can you give this migration file a `--name`?
please add reverse_sql=migrations.RunSQL.noop so this migration can be reversed

Reply:
Yup yup, done!